dvc/fs/fsspec_wrapper.py

```diff
@@ -29,6 +29,12 @@ def _with_bucket(self, path):
     def _strip_bucket(self, entry):
         return entry
 
+    @staticmethod
+    def _get_kwargs_from_urls(urlpath):
```

Comment:
fsspec also have this methods `_get_kwargs_from_urls`, but
1. it is a private method.
2. not have been implemented in all of the `fs`.